packages/attestation-service/src/requestHandlers/get_attestation.ts

```diff
@@ -87,10 +96,35 @@ class GetAttestationRequestHandler {
 }
 
 export async function handleGetAttestationRequest(
-  _req: express.Request,
+  req: express.Request,
   res: express.Response,
   getRequest: GetAttestationRequest
 ) {
+  const {
+    headers: { authentication },
+  } = req
+  if (!authentication) {
+    respondWithError(res, 401, 'Missing authentication')
+    return
+  }
+
+  if (getRequest.securityCode) {
+    const typedData = buildSecurityCodeTypedData(getRequest.securityCode)
+    const accounts = await useKit((kit) => kit.contracts.getAccounts())
+    const [walletAddress, dekAddress] = await Promise.all([
+      accounts.getWalletAddress(getRequest.account),
+      accounts.getDataEncryptionKey(getRequest.account).then((x) => x && publicKeyToAddress(x)),
+    ])
+
+    const validSignature = [getRequest.account, walletAddress, dekAddress]
+      .filter(Boolean)
```

Comment:
What is this boolean checking here? That it's not null?

Reply:
Yeah, common way to ensure only truthy values. The dek can be null, but I could also potentially check for the 0x0000... address

Reply:
Yeah I think by default it will return `0x0` which if I interpret the line below correctly, would lead to an error?